### ISSUE TITLE
tns doctor/info says "updates are available" while tns update command don't see any

### ISSUE THREAD

**Issue by funder7** [NativeScript/NativeScript#8703]
**Environment**
Provide version numbers for the following components (information can be retrieved by running `tns info` in your project folder or by inspecting the `package.json` of the project):
 - CLI: 6.7.8
 - Cross-platform modules:
 - Android Runtime: 6.5.0
 - iOS Runtime: 6.5.0
 - Plugin(s):

**Describe the bug**
Launching the commands ```tns doctor``` or ```tns info``` in order to check the local configuration, they reported some updates for android and ios modules. Launching ```tns update``` did not update the project.

**To Reproduce**
1) Create a new code-sharing project with ```ng new --collection=@nativescript/schematics my-shared-app --shared --style=scss```
2) Open package.json file and set tns-ios/android versions in nativescript object to `6.5.0` (both) 
3) Make sure that these libraries are downgraded (delete ```node_modules``` and run ```npm i``` ?)
4) Execute ```tns doctor``` OR ```tns info``

**Expected behavior**
Run ```tns-update``` and have libraries updated

**Additional context**
![immagine](https://user-images.githubusercontent.com/12503478/87168191-657d8d80-c2ce-11ea-8b11-a18f0477e009.png)

**Comment by NathanWalker** [comment]
Thanks for this @funder7 - we'll take a look in the cli 👍

**Comment by jessorlisa** [comment]
Still an issue with latest versions and `ns doctor/update`:

```
jessorlisa % ns doctor
✔ Getting environment information 

No issues were detected.
✔ Your ANDROID_HOME environment variable is set and points to correct directory.
✔ Your adb from the Android SDK is correctly installed.
✔ The Android SDK is installed.
✔ A compatible Android SDK for compilation is found.
✔ Javac is installed and is configured properly.
✔ The Java Development Kit (JDK) is installed and is configured properly.
✔ Xcode is installed and is configured properly.
✔ xcodeproj is installed and is configured properly.
✔ CocoaPods are installed.
✔ CocoaPods update is not required.
✔ CocoaPods are configured properly.
✔ Your current CocoaPods version is newer than 1.0.0.
✔ Python installed and configured correctly.
✔ The Python 'six' package is found.
✔ Xcode version 12.0.1 satisfies minimum required version 10.
✔ Getting NativeScript components versions information...
✔ Component nativescript has 7.0.8 version and is up to date.
⚠ Update available for component @nativescript/core. Your current version is 7.0.3 and the latest available version is 7.0.8.
✔ Component @nativescript/ios has 7.0.0 version and is up to date.
✔ Component @nativescript/android has 7.0.0 version and is up to date.
jessorlisa % ns update
This project is up to date.
```
